README.rst

```diff
@@ -0,0 +1,76 @@
+Example: Basic Sphinx project for Read the Docs
+===============================================
+
```

Comment:
```suggestion

.. This page uses absolute links for most things so it renders properly on GitHub

```

As we discussed 👍

Reply:
Thanks! I already wrote almost the same while we were in the call but forgot to push -- merging with your sentence